miden-lib/asm/miden/kernels/tx/prologue.masm

```diff
@@ -567,6 +567,7 @@ end
 #!      block_num,
 #!      SUB_HASH,
 #!      NOTE_ROOT,
+#!      is_authenticated,
```

Comment:
I believe `is_authenticated` flag comes before `SUB_HASH` and `NOTE_ROOT` and these words are conditional on the the flag being set.